addons/mrp/views/mrp_routing_views.xml

```diff
@@ -6,11 +6,15 @@
             <field name="name">mrp.routing.workcenter.tree</field>
             <field name="model">mrp.routing.workcenter</field>
             <field name="arch" type="xml">
-                <tree string="Routing Work Centers">
-                    <field name="sequence" widget="handle"/>
+                <tree string="Routing Work Centers" multi_edit="1">
+                    <field name="sequence" widget="handle" attrs="{'column_invisible': [('parent.id', '=', False)]}"/>
                     <field name="name"/>
+                    <field name="bom_id"/>
```

Comment:
Modify this tree view, modify the `operations` tree view of the BoM form. Then I think we should **not** show the `bom_id` in that case, maybe with the `bom_id_invisible`  from context (like in form) + column_invisible?